awsm/framework/framework.py

```diff
@@ -277,12 +281,6 @@ def parse_folder_structure(self):
             print('No base path to drive given. Exiting now!')
             sys.exit()
 
-        self.basin = self.config['paths']['basin']
-        self.water_year = utils.water_day(self.start_date)[1]
-        self.project_name = self.config['paths']['project_name']
-        self.project_description = self.config['paths']['project_description']
-        self.folder_date_style = self.config['paths']['folder_date_style']
-
```

Comment:
These variables where never used outside of reading them once when needed and hence I remove them as class properties and read it straight from the config when needed.